### DISCUSSION TITLE
Why is exploitability in percent and what is it in regard to?

### DISCUSSION THREAD

**Discussion by snickerton** [b-inary/postflop-solver#43] [Q&A]
Heyo,

I noticed that exploitability is measured in percent, is this with regard to the starting pot? blinds? stack sizes?

I understand that max exploitability strategy expected value is just the EV if playing at nash equilibrium, however I've always been used to EV being expressed in terms of blinds or raw monetary amount. Haven't been able to find a clear answer digging in the code how exactly this EV becomes a percent.

Thanks in advance!

**Comment by b-inary** [comment]
In this library, exploitability is measured as an absolute value (raw chip amount), but GUI applications (WASM Postflop and Desktop Postflop) display exploitability as a percentage of the starting pot specified in the tree configuration. I think this is the same as other typical GTO solvers, such as PioSOLVER and GTO+.
